sprint0/sprint0/LeftIdleLinkSprite.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace sprint0
+{
+    class LeftIdleLinkSprite : ISprite
+    {
+        public Texture2D Texture { get => texture; set => texture = value; }
+        public Vector2 Location { get => location; set => location = value; }
+
+        private Texture2D texture;
+        private Vector2 location;
+
+        public LeftIdleLinkSprite(Texture2D texture, Vector2 location)
+        {
+            this.texture = texture;
+            this.location = location;
+        }
+
+        public void Draw(SpriteBatch spriteBatch)
+        {
+            Rectangle destination = new Rectangle((int) location.X, (int) location.Y, 16, 16);
```

Comment:
not sure if we need to specify drawing dimensions if the source is already 16 x 16? anyways this doesn't matter too much

Reply:
This is the best way I found to flip a sprite since the sheet only has Link facing right